kernels/volk/volk_32fc_s32f_magnitude_16i.h

```diff
@@ -225,10 +225,10 @@ volk_32fc_s32f_magnitude_16i_a_sse(int16_t* magnitudeVector, const lv_32fc_t* co
     // Arrange in q1q2q3q4 format
     qValue = _mm_shuffle_ps(cplxValue1, cplxValue2, _MM_SHUFFLE(3,1,3,1));
 
-    iValue = _mm_mul_ps(iValue, iValue); // Square the I values
-    qValue = _mm_mul_ps(qValue, qValue); // Square the Q Values
+    volatile __m128 iValue2 = _mm_mul_ps(iValue, iValue); // Square the I values
+    volatile __m128 qValue2 = _mm_mul_ps(qValue, qValue); // Square the Q Values
```

Comment:
Looks good to me. Though, I thought that MSVC fails to compile because of the `volatile` keyword. I wonder how that PR changes that situation and how that PR might change our CI.